packages/material-ui-lab/src/TreeView/TreeView.d.ts

```diff
@@ -34,12 +34,16 @@ export interface TreeViewPropsBase
    */
   expanded?: string[];
   /**
-   * Callback fired when tree items are expanded/collapsed.
-   *
+   * Callback fired when tree items are expanded/collapsed
    * @param {object} event The event source of the callback.
-   * @param {array} nodeIds The ids of the expanded nodes.
-   */
-  onNodeToggle?: (event: React.ChangeEvent<{}>, nodeIds: string[]) => void;
+   * @param {string[]} nodeIds The ids of the expanded nodes.
+   * @param {'Keyboard' | 'IconClick' | 'LabelClick'} reason The reason for the expansion / collapse.
```

Comment:
The change to resolveType in generateMarkdown.js allows for string literal union types